ui/src/api/items/item.js

```diff
@@ -43,6 +43,7 @@ class QueryBuilder {
     this.statusQuery = [];
     this.barcodeQuery = [];
     this.availableForCheckoutQuery = [];
+    this.notOnLoanQuery = [];
```

Comment:
Does it make sense to use the full name instead of `notOnLoanQuery`, `notItemOnLoanQuery`?

Reply:
this will be changed for something more user friendly after a discussion I had with Nico